### ISSUE TITLE
Option (remap) to specify packaging

### ISSUE THREAD

**Issue by vhusainov** [lburgazzoli/gradle-karaf-features-plugin#22]
Please, add an option to specify packaging. I've got a feature which contains war.

For now, I'm using following workaround:

```groovy
bundle("artifact") {
  remap.version = "$remap.version/war"
}
```

**Comment by lburgazzoli** [comment]
How do you add WARs ? I mean something like 

            compile "org.apache.activemq:activemq-web-console:5.12.1@war"

**Comment by vhusainov** [comment]
I forgot to tell that in my case this is a project dependency, that's why I'm using remap...

**Comment by lburgazzoli** [comment]
Can you please check if the latest 2.9.7-SNAPSHOT does what you need ?

```groovy
bundle("group:name") {
    type = 'war'
}
```

If the bundle is already a war, it should not be required

**Comment by lburgazzoli** [comment]
@vhusainov have you had any time to test it ?

**Comment by vhusainov** [comment]
Sorry for delay. Now it works fine, but plugin could detect packaging automatically (for project dependencies). That is how I do it in my code:
```
if (dep.dependencyProject.plugins.hasPlugin(WarPlugin)) {
  type = 'war';
}
```

**Comment by lburgazzoli** [comment]
This is for artifacts generated by the project but not for dependencies, I need to improve this area

**Comment by lburgazzoli** [comment]
Can you please close this if fixed by PR #25 ?
